src/keyszer/output.py

```diff
@@ -127,8 +127,12 @@ def send_combo(self, combo):
             pressed_mod_keys.append(key)
 
         # normal key portion of the combo
+        # pre keystroke delay can be a little shorter than post
+        sleep_ms(THROTTLE_DELAYS['keystroke_delay_ms'] / 1.5)
```

Comment:
Not sure I like this extra delay hidden from the user why thinks they are defining `* 1` multiple...

Reply:
> The Unicode delay is also doubled around the “Enter” keystroke, since that was what seemed to work best after a lot of testing. So a value of 10 is actually 20 ms of delay.

You mean before and after?  That's a docs issue.. the delay is 10ms, just repeated twice.

---

> Or I can split both delays into “pre” and “post” and present them all in the API.

I think you're presuming a lot from your individual test case... but in any case if settings like before 50, after 75 "seem good" then I'd just have two different knobs and tell people in the docs/wiki that if they are having problems here are some settings they might want to try.  I don't think we have nearly enough data to start hard coding delay ratios into the core code.

Reply:
> it just happens to be repeated twice

But the end result is a longer delay than the user might be expecting, so kind of the same thing. So splitting it would make it more obvious what's happening. 

I think you're right on the static ratio being presumptuous. Although it seemed to work best in my situation it could be different in another situation. Maybe for somebody it will work better to have a longer `pre` and they can get away with a shorter `post`, or just a really long `post`. After all, we still don't have a clue how or why this timing mismatch even happens. 

Alright, I'll split up at least the main keystroke delay into a `pre` and `post`. But I think the Unicode delay should also be split. The same argument could be made that in another situation the user may get better results from having a different ratio before and after the "Enter" keystroke, or even from disabling one or the other Unicode delays. 

I'll also try to prepare some info to be added to the Readme about what values and ratios are most likely to work, to make using the API less painful.

Reply:
> user may get better results from having a different ratio before and after the "Enter" keystroke

But if we have no real evidence of that yet then let's keep the single setting for simplicity.  And explain in the docs it the delay is inserted before and after.

Reply:
OK, that should do it.